frontend/src/components/EditRecipe/EditRecipeForm.css

```diff
@@ -62,10 +69,20 @@
   border: none;
 }
 
+.st-search-input {
+  font-size: 25px;
+  font-weight: 200;
+}
+
 .form-dropdown{
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: space-between;
   margin-bottom: 50px;
 }
+
+textarea {
```

Comment:
We should use a class here, as I don't think we want to style every `textarea` on the site.